.github/workflows/ci-test.yml

```diff
@@ -94,15 +95,14 @@ jobs:
             echo "compatibility suite is enabled."
           fi
           echo "::set-output name=need-compat::$(echo $res)";
-      - id: environment
-        name: Which environment should we use for secrets.
+      - id: any-changes-found
+        name: determine if there are any files listed in the CHANGES_CHANGED_FILE_OUTPUTFILE.
         run: |
-          output=None
-          if ${{ github.event_name == 'push' }}; then
-            echo "will use sccache environment for rust builds";
-            output=Sccache
+          res=true
+          if [[ ! -f "$CHANGES_CHANGED_FILE_OUTPUTFILE" ]] || [[ "$(cat "$CHANGES_CHANGED_FILE_OUTPUTFILE" | wc -l)" == 0 ]]; then
+            res=false;
           fi
-          echo "::set-output name=name::$(echo $output)";
+          echo "::set-output name=changes-found::$(echo $res)";
```

Comment:
Environment was never used, and I should have deleted it a while back.   Now there is a test to see if any files changed in a pull request.   If any changed, output true.   Bors rebase on land will result in prs firing with no changes.